fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/TextField.kt

```diff
@@ -0,0 +1,217 @@
+package com.microsoft.fluentui.tokenized.controls
+
+import androidx.compose.foundation.LocalIndication
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.runtime.*
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.style.TextDirection
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.core.R
+import com.microsoft.fluentui.icons.SearchBarIcons
+import com.microsoft.fluentui.icons.searchbaricons.Dismisscircle
+import com.microsoft.fluentui.theme.token.FluentIcon
+import com.microsoft.fluentui.theme.token.Icon
+import com.microsoft.fluentui.theme.token.controlTokens.DividerInfo
+import com.microsoft.fluentui.theme.token.controlTokens.DividerTokens
+import com.microsoft.fluentui.theme.token.controlTokens.TextFieldInfo
+import com.microsoft.fluentui.theme.token.controlTokens.TextFieldTokens
+import com.microsoft.fluentui.tokenized.divider.Divider
+
+@Composable
+fun TextField(
+    value: String,
+    onValueChange: ((String) -> Unit),
+    modifier: Modifier = Modifier,
+    hintText: String? = null,
+    label: String? = null,
+    assistiveText: String? = null,
+    rightAccessoryText: String? = null,
+    errorString: String? = null,
+    leftPrimaryIcon: ImageVector? = null,
```

Comment:
lets rename left and right as trailing and leading. since rtl is supported it will be confusing

Reply:
Done